lib/exercism/five_a_day_count_exercises.rb

```diff
@@ -0,0 +1,33 @@
+class Five
+  def initialize
+    user_id = user.id
+    five_a_day_count = 5
+  end
+
+  def exercises_list
+    @exercises_list ||= ActiveRecord::Base.connection.execute(exercises_list_sql).to_a
+  end
+
+private
+
+  def exercises_list_sql(user_id, five_a_day_count)
```

Comment:
Yep, all of this got moved to user.rb, this is definitely not the most recent code.